assignments/array-methods.js

```diff
@@ -56,28 +57,73 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+
+runners.forEach(element => {
+    fullName.push(`${element.first_name} ${element.last_name}`)
+})
+
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
-// The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
-let allCaps = [];
+// The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and 
+// log the result
+// let allCaps = [];
+
+let allCaps = runners.map(element => element.first_name.toUpperCase())
+
 console.log(allCaps); 
 
 // ==== Challenge 3: Use .filter() ====
-// The large shirts won't be available for the event due to an ordering issue.  Get a list of runners with large sized shirts so they can choose a different size. Return an array named largeShirts that contains information about the runners that have a shirt size of L and log the result
-let largeShirts = [];
+// The large shirts won't be available for the event due to an ordering issue.  Get a list of runners with large sized shirts so they can choose a different size. 
+// Return an array named largeShirts that contains information about the runners that have a shirt size of L and log the result
+// let largeShirts = [];
+
+let largeShirts = runners.filter(element => element.shirt_size === "L")
+
 console.log(largeShirts);
 
+
+
+
 // ==== Challenge 4: Use .reduce() ====
 // The donations need to be tallied up and reported for tax purposes. Add up all the donations into a ticketPriceTotal array and log the result
-let ticketPriceTotal = [];
+// let ticketPriceTotal = [];
+
+
+// **WORKING**
+function adder(acc, currentPerson) {
```

Comment:
Huh. Kind of a different way to go about it but I'm glad it's working. Although part of the challenge was to use .reduce(), it never said you couldn't. Nicely done.